src/sdam/monitor.ts

```diff
@@ -458,23 +495,30 @@ export class RTTPinger {
   /** @internal */
   [kCancellationToken]: CancellationToken;
   /** @internal */
-  [kRoundTripTime]: number;
-  /** @internal */
   [kMonitorId]: NodeJS.Timeout;
+  /** @internal */
+  monitor: Monitor;
   closed: boolean;
+  /** @internal */
+  latestRTT?: number;
 
-  constructor(cancellationToken: CancellationToken, options: RTTPingerOptions) {
+  constructor(monitor: Monitor, cancellationToken: CancellationToken, options: RTTPingerOptions) {
     this.connection = undefined;
     this[kCancellationToken] = cancellationToken;
-    this[kRoundTripTime] = 0;
     this.closed = false;
+    this.monitor = monitor;
+    this.latestRTT = monitor.latestRTT;
```

Comment:
Done in accordance with spec: https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.md#calculation-of-average-round-trip-times